docs/edge.rst

```diff
@@ -65,17 +65,31 @@ Kubernetes cluster, one that meets the following requirements:
 * It must contain at least three nodes (for high availability).
   You can also run development workloads on a single-node cluster. Note,
   however, that you will only be able to provision single-node CrateDB
-  "clusters";
+  clusters;
 
-* Sufficient CPU per node to run the CrateDB Cloud software stack and the OS
-  (we recommend at least 4 CPU cores for reliable performance);
+* Sufficient CPU per node to run the CrateDB Cloud software stack (we recommend
+  at least 4 CPU cores for reliable performance) as well as sufficient compute
+  to run the CrateDB instances desired. We recommend the use of a K8s
+  autoscaler.
 
-* A Kubernetes version > 1.15 and < 1.22 (support for Kubernetes versions 1.22+
-  is coming in the near future);
+* A Kubernetes version 1.20, 1.21, or 1.22;
 
-* A Kubernetes load balancer for accessing CrateDB Clusters;
+* A Kubernetes load balancer for accessing CrateDB clusters;
```

Comment:
We do **NOT** require a loadbalancer for accessing crateDB as we are using _cloudflare tunnels_ to work around networking issues.